src/service/viewport-impl.js

```diff
@@ -652,8 +671,11 @@ export class ViewportBindingDef {
   /**
    * Updates binding with the new padding.
    * @param {number} unusedPaddingTop
+   * @param {boolean|undefined} unusedOptUpdateScrollPos
```

Comment:
It seems `updateScrollPos` is useless, since it's only specified when `lastPaddingTop` is also. So, we could just check to see if `lastPaddingTop` is passed.